homeassistant/components/dlna_dmr/config_flow.py

```diff
@@ -205,7 +204,7 @@ async def async_step_import_turn_on(
         self._set_confirm_only()
         return self.async_show_form(step_id="import_turn_on", errors=errors)
 
-    async def async_step_ssdp(self, discovery_info: DiscoveryInfoType) -> FlowResult:
+    async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowResult:  # type: ignore[override]
```

Comment:
`# type: ignore[override]` is temporary (required by mypy), and will be removed in part 2